src/aero/actuator/ActuatorFunctors.C

```diff
@@ -91,15 +91,21 @@ SpreadForceInnerLoop::operator()(
   actuator_utils::compute_distance(
     3, nodeCoords, pointCoords.data(), &distance[0]);
 
-  const double gauss =
-    actuator_utils::Gaussian_projection(3, &distance[0], epsilon.data());
-
-  for (int j = 0; j < 3; j++) {
-    projectedForce[j] = gauss * pointForce(j);
-  }
+  //Check distance between actuator point and element centroid. Only needed if singlePointCoarseSearch_==False
+  //auto epsilonRadius = 
+  //  Kokkos::subview(actBulk_.searchRadius_.view_host(), pointId, Kokkos::ALL);
+  auto epsilonRadius = actBulk_.searchRadius_.h_view(pointId);
+  if (std::sqrt(distance[0]*distance[0] + distance[1]*distance[1] + distance[2]*distance[2]) < epsilonRadius) {
```

Comment:
I've added a check based on the distance from the actuator point to the element centroid before evaluating the exponential, which uses the predefined values in searchRadius_. I think this check is only needed if singlePointCoarseSearch_=false, right? If so, I can add another conditional to the if statement

Reply:
It shouldn't make a difference. The search radius is epsilon for the old case. I think profiling is the only way to know if this would slow the old case down.